meilisearch/landing
# Create the `our story` block

# Pull Request

## Related issue
Closes #479

## What does this PR do?
- Addition of a new block `Our Story`
- Addition of the corresponding story for Storybook

## How to test

Go to [https://landing-git-createblockourstory-meili.vercel.app/about-us](https://landing-git-createblockourstory-meili.vercel.app/about-us) and check the `Our Story` block .
You can also go to Storybook (`yarn storybook`)

## PR checklist
Please check if your PR fulfills the following requirements:

Thank you so much for contributing to Meilisearch!
